### ISSUE TITLE
Expanding caret is nearly invisible

### ISSUE THREAD

**Issue by filiptronicek** [steveseguin/vdo.ninja#716]
The caret that expands the features of obs.ninja is very hard to see under any outside light or any disability. It fails all of the WCAG standards. I think this should be changed, it shouldn't be that hard hopefully.
Contrast checker: https://webaim.org/resources/contrastchecker/?fcolor=141926&bcolor=292D35

Screenshot:
![screenshot of obs.ninja](https://user-images.githubusercontent.com/29888641/107534804-7a056980-6bc0-11eb-8ca5-ebd855c1e396.png)

**Comment by filiptronicek** [comment]
A design proposal (with the same colors as the cards above)
![image](https://user-images.githubusercontent.com/29888641/107535065-b46f0680-6bc0-11eb-8d06-b89459f2fd32.png)

A proposal with a darker color (#999)
![image](https://user-images.githubusercontent.com/29888641/107535370-f5671b00-6bc0-11eb-9502-eca77d76ecac.png)

**Comment by steveseguin** [comment]
Thank you kindly for the suggestion.

I purposely made it hard to see as some of those features I consider experimental; at least at the time

I'm on the fence a bit about removing those options completely and trying to find ways to include them in other ways.

I welcome feedback

**Comment by filiptronicek** [comment]
I figured that it is probably on purpose, but I think that there is a better way to approach experimental features. 

What I would do, is to create a simple settings page on the top right of the home screen that would allow users to see "beta" or experimental features. If we use JS's [localstorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) (which looks like it is very strict on what it can be accessed from), there would be AFAIK no privacy or security tradeoff. 

This would make it more simple (no caret) for regular visitors and more easily accessible (more visible caret) for people who want to try the new stuff out.
